Bottom: a6edd10c037680b3b839ff2efcea818cded96a77
Top:    e248d96887d61d1cb8d7ab2b183ac13c5d93b3ba
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-23 18:48:01 -0400

Refresh of mpc5200-dma

---

diff --git a/sound/soc/fsl/mpc5200_psc_i2s.c b/sound/soc/fsl/mpc5200_psc_i2s.c
index 2186f4b..f3c1fad 100644
--- a/sound/soc/fsl/mpc5200_psc_i2s.c
+++ b/sound/soc/fsl/mpc5200_psc_i2s.c
@@ -261,6 +261,7 @@ static int __devexit psc_i2s_of_remove(struct of_device *op)
 /* Match table for of_platform binding */
 static struct of_device_id psc_i2s_match[] __devinitdata = {
 	{ .compatible = "fsl,mpc5200-psc-i2s", },
+	{ .compatible = "fsl,mpc5200b-psc-i2s", },
 	{}
 };
 MODULE_DEVICE_TABLE(of, psc_i2s_match);
